samples/bluetooth/advertiser/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
```

Comment:
CMake 3.20 is now the minimum version required by Zephyr.
```suggestion
cmake_minimum_required(VERSION 3.20)
```

Reply:
Changed the cmake minimum version requirement.